Gemfile

```diff
@@ -9,6 +9,7 @@ ruby '3.1.0'
 gem 'byebug'
 gem 'byebug-dap', '~> 0.1.4'
 gem 'devise'
+gem 'factory_bot_rails'
```

Comment:
I think we will not be using `factory_bot_rails` on production, so it can be moved into `development, test` group